SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -2174,16 +2138,7 @@ void USpatialReceiver::ResolvePendingOperations(UObject* Object, const FUnrealOb
 	UE_LOG(LogSpatialReceiver, Verbose, TEXT("Resolving pending object refs and RPCs which depend on object: %s %s."), *Object->GetName(), *ObjectRef.ToString());
 
 	ResolveIncomingOperations(Object, ObjectRef);
-	if (Object->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_Singleton) && !Object->IsFullNameStableForNetworking())
-	{
-		// When resolving a singleton, also resolve using class path (in case any properties
-		// were set from a server that hasn't resolved the singleton yet)
-		FUnrealObjectRef ClassObjectRef = FUnrealObjectRef::GetSingletonClassRef(Object, PackageMap);
-		if (ClassObjectRef.IsValid())
-		{
-			ResolveIncomingOperations(Object, ClassObjectRef);
-		}
-	}
```

Comment:
this feels redundant now, need someone to reason with me whether that's true or not

Reply:
This was because you could refer to a singleton using a stably named path (which was actually the class path), and the bool in the unreal obj ref. As you've removed that now, this isn't necessary.

Reply:
Do we need this back now?

Reply:
I believe so, because any object we're no longer referring too as singleton, could have two unrealobjrefs. One, for non-auth servers prior to knowing the entity id. And the other when we have the entity id.

Reply:
Since deciding to get rid of them altogether, this should stay deleted

Reply:
But we're still using `use_class_path_to_load_object`?